controllers/apply/awards-for-all/fields.js

```diff
@@ -1372,10 +1372,6 @@ module.exports = function fieldsFor({ locale, data = {}, flags = {} }) {
                 en: 'Full name of main contact',
                 cy: 'Enw llawn y prif gyswllt'
             }),
-            explanation: localise({
```

Comment:
Removing this on the basis that:

- We already validate the address against a UK-only postcode, so the help text is redundant
- It doesn't always apply (i.e. if you are a statutory body where we don't ask for address)

Have queried this on the related card. If we _do_ need to keep it this help text it should go alongside the address field instead.

Reply:
Moved this out to a task in the backlog